src/rulesets/oas/index.json

```diff
@@ -114,7 +114,6 @@
     "example-value-or-externalValue": {
       "description": "Example should have either a `value` or `externalValue` field.",
       "recommended": true,
-      "severity": "warn",
```

Comment:
@philsturgeon Similarly to "error", this was making one test fail. I defaulted to drop the line and rely on that fact that "warning" severity is the default one.

Reply:
This is great! You can remove the line in the test where I had to override the severity string with the typescript constant. Those were getting annoying.

Reply:
Most of the additions I could drop no longer exist